gifts.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Gifts | Armani</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body style="background-image:url(images/INSTORCOLLECTION.webp); background-size:cover; background-position-y:150%; background-repeat:no-repeat"
+>
+    <nav class="navigation-bar">
+      <a href="#">
+        <img class="logo" src="images/armani-logo.png" alt="armani-logo"
+      /></a>
```

Comment:
Nu te mai baza pe formatarea de la VSCode.

```suggestion
        <img class="logo" src="images/armani-logo.png" alt="armani-logo" />
      </a>
```